01_Core/week04/ecommerce/contact/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+
+<html>
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width">
+		<title>Stardust Designs || Contemporary Furniture</title>
+		<link rel="stylesheet" href="../stylesheets/normalize.css">
+		<link rel="stylesheet" href="../stylesheets/style.css">
+		<script src="validation.js"></script>
+	</head><!--/head-->
+	<body>
+		<header>
+			<nav class="main-nav">
+				<ul class="nav__ul h-marg-0">
+					<li><img src="../images/logo_banner.png" class="logo"></li>
+					<li><a href="../index.html">Home</a></li>
+					<li><a href="../products/index.html">Products</a></li>
+					<li><a href="../contact/index.html">Contact</a></li>
+					<li><a href="../search/index.html">Search</a></li>
+				</ul>
+				<!--.footer__ul-->
+			</nav>
+			<!-- header navigation with Home, Product, Search, Contact, Cart -->
+		</header>
+		<main>
+			<h1 class="h-padtop">Drop us a line!</h1>
+			<p>To get in touch, please fill out the contact form below. Fields with a red asterisk are required. </p>
+
+			<form method="GET" name="contact_information" class="contact-form">
```

Comment:
You won't be using it yet, but form tags require an action property. This specifies the location where the data will be submitted. Also, GET should be lowercase.